code/game/objects/effects/anomalies.dm

```diff
@@ -2,6 +2,18 @@
 /// Chance of taking a step per second
 #define ANOMALY_MOVECHANCE 45
 
+/// Lists for zones and bodyparts to swap and randomize
+#define ANOMALY_BIOSCRAMBLER_ZONES list(BODY_ZONE_HEAD, BODY_ZONE_CHEST, BODY_ZONE_L_ARM, BODY_ZONE_R_ARM, BODY_ZONE_L_LEG, BODY_ZONE_R_LEG)
+#define ANOMALY_BIOSCRAMBLER_ZONE_CHEST typesof(/obj/item/bodypart/chest) - typesof(/obj/item/bodypart/chest/robot)
+#define ANOMALY_BIOSCRAMBLER_ZONE_HEAD typesof(/obj/item/bodypart/head) - typesof(/obj/item/bodypart/head/robot) - typesof(/obj/item/bodypart/head/ipc)
+#define ANOMALY_BIOSCRAMBLER_ZONE_L_LEG typesof(/obj/item/bodypart/l_leg) - typesof(/obj/item/bodypart/l_leg/robot) - typesof(/obj/item/bodypart/l_leg/ipc)
+#define ANOMALY_BIOSCRAMBLER_ZONE_R_LEG typesof(/obj/item/bodypart/r_leg) - typesof(/obj/item/bodypart/r_leg/robot) - typesof(/obj/item/bodypart/r_leg/ipc)
+#define ANOMALY_BIOSCRAMBLER_ZONE_L_ARM typesof(/obj/item/bodypart/l_arm) - typesof(/obj/item/bodypart/l_arm/robot) - typesof(/obj/item/bodypart/l_arm/ipc) 
+#define ANOMALY_BIOSCRAMBLER_ZONE_R_ARM typesof(/obj/item/bodypart/r_arm) - typesof(/obj/item/bodypart/r_arm/robot) - typesof(/obj/item/bodypart/r_arm/ipc)
+#define ANOMALY_BIOSCRAMBLER_ZONE_ORGANS typesof(/obj/item/organ) - typesof(/obj/item/organ/brain) - typesof(/obj/item/organ/body_egg) - /obj/item/organ/alien/eggsac - /obj/item/organ/zombie_infection - /obj/item/organ/cyberimp - /obj/item/organ/heart/cursed
```

Comment:
Use typesof(/obj/item/organ/cyberimp) instead so it removes all the types of implants

Reply:
i did

Reply:
'- /obj/item/organ/cyberimp`

Reply:
oh wait shit